spec/requests/green_lanes/applicable_exemptions_controller_spec.rb

```diff
@@ -27,8 +27,8 @@
     let(:make_request) do
       post green_lanes_applicable_exemptions_path, params: {
         exemptions: {
-          category_assessment_34: ['', 'Y997'],
-          category_assessment_82: ['', 'none'],
+          category_assessment_a6b633a7b098132ec45c036d0e14713a: ['', 'Y997'],
+          category_assessment_18fcbb5b75781f8a676bd84dae9c170e: ['', 'none'],
```

Comment:
weirdly if you test it manually, it has 3 category assessments, not to :thinking: 
Maybe it's the cassette

Reply:
I think its just an old cassette, yeah. We can refresh the cassette at some point if needed